src/python/controller/controller.py

```
@@ -638,6 +749,25 @@ def __spawn_move_process(self, file_name: str):
         process.start()
         self.logger.info("Spawned move process for {} (staging -> local)".format(file_name))
 
+    def __spawn_system_delete_local(self, file_name: str):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `__spawn_system_delete_local`

Add return type annotation: `None`

(ANN202)